lambda/custom/test/tests.js

```diff
@@ -0,0 +1,866 @@
+var AWS = require('aws-sdk');
+AWS.config.update({ region: 'us-east-1' });
+const assert = require('chai').assert;
+const vax = require('virtual-alexa');
+const constants = require('../constants');
+// You need to setup the same ENV var as the Lambda function
```

Comment:
I think we should document this note in readme perhaps?

Reply:
Added. Also added an error message